File: chartview/src/main/java/com/bitvale/chartview/ChartSpinnerListener.java
package com.bitvale.chartview;

import com.bitvale.chartview.widget.chart.ChartSpinner;

/**
 * Created by Alexander Kolpakov (jquickapp@gmail.com) on 16-Mar-19
 */
public interface ChartSpinnerListener {
    void onRangeChanged(int daysBeforeFrame, int daysAfterFrame, int daysInFrame, float dx, ChartSpinner.State state);
}

File: chartview/src/main/java/com/bitvale/chartview/ChartSpinnerListener.kt
package com.bitvale.chartview

/**
 * Created by Alexander Kolpakov (jquickapp@gmail.com) on 10-Mar-19
 */
interface ChartSpinnerListener {
    fun onRangeChanged(daysBeforeFrame: Int, daysAfterFrame: Int, daysInFrame: Int, dx: Float)
}